### PR TITLE
Execute ReadInConfig before init logger

### PR DESCRIPTION
**Pull request by asdfsx**
Read config into viper before init logger.
So that `	logging.InitLog()` can read configurations from `configs.yaml`
Right now `InitLog` can not get log level parameter inside the `configs.yaml`, cause the `InitLog` is executed before viper read the `configs.yaml`

### CODE REVIEW

**Review by LaynePeng — COMMENTED** [review]
What if ReadInConfig then throw an error? The log will not work to deal with the error in L32, right?

**Comment by asdfsx** [comment]
PR just updated. The right way is to change the position where `ReadInConfig` execute, not add ReadInConfig in `InitViper` function.

File: k8s-deploy/kubefate.go [hunk 1]
```diff
@@ -27,12 +27,13 @@ import (
 
 func main() {
 	config.InitViper()
-	logging.InitLog()
 	err := viper.ReadInConfig()
 	if err != nil {
 		log.Debug().Err(err).Msg("load config.yaml error")
```
Comment (jiahaoc1993) [inline k8s-deploy/kubefate.go:32]: Could you please also replace this line with `fmt.Println()` ?

Comment (asdfsx) [reply k8s-deploy/kubefate.go:32]: @jiahaoc1993 replace `log.Debug().Err(err).Msg("load config.yaml error")` with `fmt.Println("load config.yaml error")`?

Comment (asdfsx) [reply k8s-deploy/kubefate.go]: @jiahaoc1993 I have been replace `log` with `fmt.Println`, is this what you mean?

Comment (LaynePeng) [reply k8s-deploy/kubefate.go]: Looks good to me now. @asdfsx could you please sign-off this commit? Ref: https://github.com/FederatedAI/KubeFATE/pull/369/checks?check_run_id=2083856881

**Review by jiahaoc1993 — APPROVED** [review]
LGTM

**Pull request merged**